src/control/cmd/dmg/pretty/pool.go

```diff
@@ -270,15 +233,15 @@ func poolListCreateRowVerbose(pool *control.Pool) txtfmt.TableRow {
 
 	row := txtfmt.TableRow{
 		"Label":          label,
-		"UUID":           pool.UUID,
-		"State":          pool.State,
+		"UUID":           pool.UUID.String(),
+		"State":          pool.State.String(),
 		"SvcReps":        svcReps,
-		"Disabled":       fmt.Sprintf("%d/%d", pool.TargetsDisabled, pool.TargetsTotal),
+		"Disabled":       fmt.Sprintf("%d/%d", pool.DisabledTargets, pool.TotalTargets),
```

Comment:
why the change from <component><verb> to the other way round? thought we tended to stay with the former?

Reply:
Had to choose one or the other when doing the merge, and the `PoolInfo` names had been around longest.